docs/riff.md

```diff
@@ -4,7 +4,31 @@ riff is for functions
 
 ### Synopsis
 
-<todo>
+The riff CLI combines with the projectriff system CRDs to build, run and wire
+workloads (applications and functions). This CLI is one interface to the riff
+API.
+
+Before running riff, please install the projectriff system and its dependencies.
+See https://projectriff.io/docs/getting-started/
+
+This CLI contains commands that fit into four themes:
+- builds
+- requests
+- streams
+- misc
+
+Build commands include the application and function command to define build
```

Comment:
Would be nice if we could emphasize the `application` and `function` (and all other instances of commands in the rest of help) words somehow, at least on screen. Use of color ?

Reply:
Color should only be used as a accent and never as a primary conveyance.

For references of commands we could specify them as `riff application`. This structure is a bit painful to author, but is doable.

Reply:
> Color should only be used as an accent and never as a primary conveyance.
I'm not asking for anything else. I'm also advocating for keeping the "command" word.

Not sure about your second sentence. Are you advocating for the usage of backticks? or addition of the executable name in front?